engine/admin/logging/awslogs.md

```diff
@@ -58,6 +58,17 @@ specified, the container ID is used as the log stream.
 > at a time.  Using the same log stream for multiple containers concurrently
 > can cause reduced logging performance.
 
+### awslogs-create-group
+
+Log driver will return an error by default if the log group does not exist. However, if you want to
```

Comment:
Please reword this to be a bit more concise. Something like:

However, you can set the `awslogs-create-group` to `true` to automatically create the log group as needed. The `awslogs-create-group` option defaults to `false`.

Note: Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.